mftf/2.3/debugging.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Debugging MFTF Tests
+mftf-release: 2.3.13
+---
+
+Debugging within the Magento Functional Testing Framework is helpful in identifying test bugs by allowing you to pause execution so that you may:
+
+- Examine the page.
+- Check returned data and other variables being used during run-time.
+
+This is straightforward to do once you create a basic Debug Configuration.
+
+## Prerequisites
+
+- [Xdebug]
+- PHPUnit configured for use in [PHPStorm]
+
+## Creating Debug Configuration with PHPStorm
+
+1. Download the Codeception Framework plugin for PHPStorm (`PhpStorm->Preferences->Plugins`).
+1. Click `Edit Configurations` on the configuration dropdown.
+1. Click `+` and select `Codeception` from the available types.
+1. Find the `Custom Working Directory` option and set the path to your `dev/tests/acceptance/` directory.
+
+If you get a warning `Path to Codeception for local machine is not configured.`:
+
+1. Click `Fix`, then `+`, and select `Codeception Local`.
+1. Click `...` and locate `/vendor/bin/codecept` in your Magento installation folder.
+
+At this point, you have a couple choices in how to select the test you want to debug.
+The easiest method of tagging a test for debugging is the following:
```

Comment:
You say there's a couple of choices, but only talk about one. If you want to mention only this method, then delete the "couple of choices" sentence, then add a note afterward stating other ways of selecting a test may exist.

Reply:
Got rid of the whole sentence.